pyteal/ast/subroutine.py

```diff
@@ -154,8 +148,8 @@ def _validate(
                 Parameter.POSITIONAL_OR_KEYWORD,
             ) and not (
                 param.kind is Parameter.KEYWORD_ONLY
-                and self.abi_output_arg_name is not None
-                and name == self.abi_output_arg_name
+                and self.has_abi_output
+                and name == ABIReturnSubroutine.OUTPUT_ARG_NAME
```

Comment:
```suggestion
                and name == self.OUTPUT_ARG_NAME
```
a bit more future-proof / pythonic

Reply:
Ahhh I get it. Sorry